agent/stats/types.go

```diff
@@ -53,6 +54,8 @@ type UsageStats struct {
 	NetworkStats      *NetworkStats `json:"networkStats"`
 	Timestamp         time.Time     `json:"timestamp"`
 	cpuUsage          uint64
+	// sent indicates if the stat has been sent to TACS already.
+	sent bool
```

Comment:
Personally, I'd find it easier to understand if we had this explicitly set to false to start with, wherever UsageStats is instantiated.

Reply:
sure, will do

Reply:
done